### DISCUSSION TITLE
SkyWalking UI - Listening address

### DISCUSSION THREAD

**Discussion by jarofi** [apache/skywalking#10948] [Q&A] [answered]
Hello everybody,

Is it possible to configure the SkyWalking Native UI to listen only on a specific IP address? According to the documentation, only the port can be changed in the configuration file:
`serverPort: ${SW_SERVER_PORT:-8080}`

Thank you very much in advance for your answers.

**Comment by pg-yang** [comment]
The easiest way is specifying `SW_SERVER_PORT ` via environment variable.
The configuration policy is
```
propertyKey: ${environmentKey:defaultValue}
```

**Reply by jarofi** [reply]
@pg-yang Thank you for your reply. But I'm sorry, my question was not about changing the port, but limiting it to one IP address, specifically 127.0.0.1. Or did I misunderstand your answer?

**Reply by pg-yang** [reply, marked as answer]
Oops, sorry for my mistake.   It's unsupported to specify the listening IP.  
The blow is the source code.

https://github.com/apache/skywalking/blob/a23f905ce01bd782f0e60ddec014e02a7d6b02f1/apm-webapp/src/main/java/org/apache/skywalking/oap/server/webapp/ApplicationStartUp.java#L47-L49

**Comment by jarofi** [comment]
@pg-yang Nothing happened. Unfortunately, you've confirmed my fears. I'll try putting the query in the "Ideas" category. Thank you for your help.
